facts/inventory.py

```diff
@@ -166,13 +166,21 @@ def ip6toptr(ip_address):
 
 
 def dhcp6ranges(prefix, bitmask):
-    """generates start and end IPv6 addresses for 2x DHCP ranges"""
+    """generates start and end IPv6 addresses for DHCP ranges"""
     if bitmask == 0:
-        return ["", "", "", ""]
+        return ["", ""]
     prefsplit = re.split(r"\:\:", prefix)[0]
+    # TODO: Infer this going forward
+    # Conditional for the exVmVendor vlan, matching the ipv4 /20
+    if prefix == "2001:470:f026:112::":
```

Comment:
agree with TODO, but good iterative compromise for now.

Reply:
Agreed.